abc.templates/README.md

```diff
@@ -0,0 +1,18 @@
+# GitHub Token Minter
+
+## Installation with abc CLI
+
+1. Render infrastructure template and actuate terraform.
+```
+abc templates render \
+--dest infra/github-token-minter \
+--input=automation_service_account_member=<SERVICE_ACCOUNT> \
+--input=project_id=<PROJECT_ID> \
+--input=domains=<COMMA_SEPARATED_DOMAINS> \
+--input=terraform_state_bucket=<TERRAFORM_STATE_BUCKET> \
+--input=terraform_state_prefix=<TERRAFORM_STATE_PREFIX> \
+--input=github_owner_id=<GITHUB_ORG_ID> \
+github.com/abcxyz/github-token-minter.git//abc.templates/infra?ref=<SHA>
+```
+
+At least 1 domain is required for GitHub Token Minter.
```

Comment:
Just FYI, we have a list of domains for the input to allow domain migrations, I think its safe to say a domain is required. Also, the domains input can be singular for bootstrapping.

Reply:
ack, thanks for context